tests/test_macos.py

```diff
@@ -45,3 +67,45 @@ def test_macos(params: dict[str, Any], func: str) -> None:
     expected = expected_map[func]
 
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "params",
+    [
+        pytest.param({}, id="no_args"),
+        pytest.param({"appname": "foo"}, id="app_name"),
+        pytest.param({"appname": "foo", "version": "v1.0"}, id="app_name_version"),
+    ],
+)
+@pytest.mark.parametrize(
+    "site_func",
+    [
+        "site_data_dir",
+        "site_config_dir",
+        "site_cache_dir",
+        "site_runtime_dir",
+    ],
+)
+@pytest.mark.parametrize("multipath", [pytest.param(True, id="multipath"), pytest.param(False, id="singlepath")])
+def test_macos_homebrew(mocker: MockerFixture, params: dict[str, Any], multipath: bool, site_func: str) -> None:
+    mocker.patch("sys.prefix", "/opt/homebrew/opt/python")
+
+    result = getattr(MacOS(multipath=multipath, **params), site_func)
+
+    home = str(Path("~").expanduser())
+    suffix_elements = tuple(params[i] for i in ("appname", "version") if i in params)
+    suffix = os.sep.join(("", *suffix_elements)) if suffix_elements else ""  # noqa: PTH118
+
+    expected_map = {
+        "site_data_dir": f"/opt/homebrew/share{suffix}",
+        "site_config_dir": f"/opt/homebrew/share{suffix}",
+        "site_cache_dir": f"/opt/homebrew/var/cache{suffix}",
+        "site_runtime_dir": f"{home}/Library/Caches/TemporaryItems{suffix}",
```

Comment:
Why are we testing site_runtime_dir if that's not impacted? 🤔

Reply:
Because it's a site variable, and it's worth clarifying that it's the same when Homebrew is active.